pkgs/tools/text/mdbook-epub/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, bzip2
+, CoreServices
+, nix
+}:
+
+let
+  pname = "mdbook-epub";
+  version = "unstable-2022-12-25";
+in rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "michael-f-bryan";
+    repo = pname;
+    rev = "2e1e48d0d1a1b4c1b0f866267e6666b41c598225";
+    hash = "sha256-wjn/7dv/Z2OmwvH/XaEeCz/JOvJWlMJ60q5qozzOEWY=";
+  };
+
+  cargoHash = "sha256-4oSpQUYJDK0srABZMwJ8x8jv6DOnLShXSnjLjf8c9Ac=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    bzip2
+  ] ++ lib.optionals stdenv.isDarwin [
+    CoreServices
+  ];
+
+  passthru = {
+    tests = {
+      inherit nix;
```

Comment:
Why nix?

Reply:
mdbook is a dependency of nix, that's why nix is added as a test, which is not applicable to mdbook-epub.

If you follow the linked thread closely you would find that there's currently no direct benefit from removing that darwin inherit. Just keep it as-is is fine (given that it's indeed a dependency of mdbook-epub).